recipes/mosquitto/2.x/conanfile.py

```diff
@@ -0,0 +1,82 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class Mosquitto(ConanFile):
+    name = "mosquitto"
+    license = "EPL-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://mosquitto.org"
+    description = """Eclipse Mosquitto MQTT library, broker and more"""
+    topics = ("MQTT", "IoT", "eclipse")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+                "ssl": [True, False],
+                "clients": [True, False],
+                "broker": [True, False],
+                "apps": [True, False],
+                "plugins": [True, False],
+                "cjson": [True, False],
```

Comment:
I would rename the options `ssl` and `cjson` to `with_ssl` and `with_cjson` to make clear that these will include an extra dependency.

Reply:
ok